### DISCUSSION TITLE
Presidio Analyzer is identifying tokens like 'today'/'a moment' as date_time, how to rectify to precise date_times only?

### DISCUSSION THREAD

**Discussion by himsgpt** [microsoft/presidio#1136] [Q&A]
Analyzer is recognizing 'today'/'a moment' as date or date_time, how to rectify to precise date_times only?

Analyzer is recognizing 'today'/'a moment' as date or date_time, how to rectify to precise date_times only?
 {'entity_type': 'DATE', 'score': 0.85, 'start': 80, 'entity': 'today'}, 
{'entity_type': 'DATE_TIME', 'score': 0.85, 'start': 80, 'entity': 'today'}, 
{'entity_type': 'DATE_TIME', 'score': 0.85, 'start': 1070, 'entity': 'a moment'}

**Comment by omri374** [comment]
Duplicate of #1139
